core/src/main/scala/scala/macros/core/Universe.scala

```diff
@@ -79,10 +83,12 @@ trait Universe {
   // Semantic
   // =========
   type Mirror
-  type Symbol
-  type Denotation
 
+  type Symbol
   def symName(sym: Symbol): Name
+  def symOwner(sym: Symbol): Option[Symbol]
```

Comment:
In Dotty, every symbol has an owner, except the "_root_". Maybe, we can just remove the option type, with the assumption/specification that _non-root symbols always have an owner_? That would make the usage much easier.

If the current symbol doesn't have owner, we can define it to return `null`, or we provide a query `hasOwner` on symbol.

Reply:
I agree returning Option is inconvenient for maybe 90% use cases. However `null` is not an option IMO and I'm personally not a fan of throwing exceptions. By documenting all cases in the API, users don't have to read the docs to know when/if a method can fail. Wrapping all method call in `Try` is also inconvenient.

I propose we 

1. keep this unchanged in this PR
2. open a discussion on error handling
3. refactor API to match error handling we agree on